webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -772,37 +1099,53 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
               projectId={project.id}
               selectedExperiments={selectedExperiments}
               onWidthChange={handleCompareWidthChange}>
-              <GlideTable
-                colorMap={colorMap}
-                columnWidths={settings.columnWidths}
-                comparisonViewOpen={settings.compare}
+              <DataGrid<ExperimentWithTrial, ExperimentAction, ExperimentItem>
+                columns={columns}
                 data={experiments}
-                dataTotal={isPagedView ? experiments.length : Loadable.getOrElse(PAGE_SIZE, total)}
-                formStore={formStore}
-                heatmapOn={settings.heatmapOn}
-                heatmapSkipped={settings.heatmapSkipped}
-                height={height}
+                getHeaderMenuItems={getHeaderMenuItems}
+                getRowAccentColor={getRowAccentColor}
+                hideUnpinned={settings.compare}
+                imperativeRef={dataGridRef}
+                isPaginated={isPagedView}
                 page={page}
+                pageSize={PAGE_SIZE}
                 pinnedColumnsCount={isLoadingSettings ? 0 : settings.pinnedColumnsCount}
-                project={project}
-                projectColumns={projectColumns}
-                projectHeatmap={projectHeatmap}
-                rowHeight={globalSettings.rowHeight}
-                scrollPositionSetCount={scrollPositionSetCount}
-                selectAll={selectAll}
+                renderContextMenuComponent={({
+                  cell,
+                  rowData,
+                  link,
+                  open,
+                  onComplete,
+                  onClose,
+                  onVisibleChange,
+                }) => {
+                  return (
+                    <ExperimentActionDropdown
+                      cell={cell}
+                      experiment={getProjectExperimentForExperimentItem(
+                        rowData.experiment,
+                        project,
+                      )}
+                      link={link}
+                      makeOpen={open}
+                      onComplete={onComplete}
+                      onLink={onClose}
+                      onVisibleChange={onVisibleChange}>
+                      <div />
+                    </ExperimentActionDropdown>
+                  );
+                }}
+                rowHeight={rowHeightMap[globalSettings.rowHeight as RowHeight]}
                 selection={selection}
-                sortableColumnIds={columnsIfLoaded}
                 sorts={sorts}
                 staticColumns={STATIC_COLUMNS}
+                total={Loadable.getOrElse(PAGE_SIZE, total)}
```

Comment:
fix: `isPagedView ? runs.length : Loadable.getOrElse(PAGE_SIZE, total)`

Reply:
actually should not need this for `total` since it's only used when `isPaginated` is `false`, and `DataGrid` already sets the correct value for `rows` if `isPaginated`